### PR TITLE
Private community

### PR DESCRIPTION
**Pull request by Nutomic**

### CODE REVIEW

**Comment by dessalines** [comment]
You might have forgotten to add the new files to this PR, which is why CI is failing.

**Comment by Nutomic** [comment]
Can be merged now.

File: src/types/ListCommunityPendingFollows.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// This file was generated by [ts-rs](https://github.com/Aleph-Alpha/ts-rs). Do not edit this file manually.
+
+export interface ListCommunityPendingFollows {
+  pending_only: boolean | null;
+  all_communities: boolean | null;
+  page: number | null;
+  limit: number | null;
```
[the remark this answers is not in the document]
Comment (Nutomic) [reply src/types/ListCommunityPendingFollows.ts:7]: Oops I didnt update the PR after fixing those errors in the backend. Fixed in a moment.

**Review by dessalines — APPROVED** [review]

**Review by SleeplessOne1917 — APPROVED** [review]

**Comment by dessalines** [comment]
I'll handle the merge conflicts with this one.

**Pull request merged**